components/docs-chef-io/content/automate/ha_healthcheck.md

```diff
@@ -21,78 +21,141 @@ This page includes commands that can be executed for the Chef Automate cluster p
 
 - Get the Automate HA cluster Information 
-  ```cmd
+  
+```cmd
     chef-automate info 
-  ```
+```
 
 - Post Deployment, run smoke test cases on Automate HA cluster, please run the command from bation node
-  ```cmd
+  
+```cmd
     chef-automate test --full 
-  ```
+```
 
 - Validate the cluster but skip "chef-automate diagnostic run" when performing the smoke tests
+  
   ```cmd
     chef-automate test  
   ```
 
 - Run the smoke test on specific cluster 
+  
   ```cmd
     chef-automate test automate
     chef-automate test chef_server
     chef-automate test opensearch
     chef-automate test postgresql 
   ```
 
-
 - To get the status of the cluster, run the command from bastion node. 
+  
   ```cmd
     chef-automate status 
   ```
 
 - Check the service status on frontend nodes (Automate), ssh to the frontend node.
-   ```cmd 
+
+```cmd
     chef-automate ssh --hostname a2
     chef-automate status
-  ```
+```
 
 - Check the service status on frontend nodes (Chef Infra Server), ssh to the frontend node.
-   ```cmd 
+
+```cmd
     chef-automate ssh --hostname cs
     chef-automate status
-  ```
+```
 
 - Check the service status on backend nodes (Postgres nodes), ssh to the backend node.
-  ```cmd 
+  
+```cmd
    chef-automate ssh --hostname pg
    hab svc status
-  ```
+```
 
 - Check the service status on backend nodes (Opensearch nodes), ssh to the backend node.
-  ```cmd 
+  
+```cmd
     chef-automate ssh --hostname os
     hab svc status
-  ```
+```
 
-- Patch a config to the Front end nodes 
-  - add the config to the location `/hab/a2_deploy_workspace/config/<automate.toml>`
-  ``` cmd
-    chef-automate config patch /hab/a2_deploy_workspace/config/automate.toml
-  ```
+
+
+- Patch a config to the Front end nodes (Automate)
+  - create a config file `automate.toml`
+  
+``` cmd
+    chef-automate config patch automate.toml --automate
+```
+sorthands for --automate is --a2 and -a
+
+- Patch a config to the Front end nodes (Chef Server)
+  - create a config file `chefserver.toml`
+  
+``` cmd
+    chef-automate config patch chefserver.toml --chef_server
+```
+sorthands for --chef_server is --cs and -c
+
+- Patch a config to the all Front end nodes (Chef Server + Automate)
+  - create a config file `frontend.toml`
+  
+``` cmd
+    chef-automate config patch frontend.toml --frontend
+```
+sorthands for --chef_server is --fe and -f
+
+- Patch a config to the Back end nodes (Open Search)
+  - create a config file `opensearch.toml`
+
+``` cmd
+    chef-automate config patch opensearch.toml --opensearch
+```
+sorthands for --opensearch is --os and -o
+
+- Patch a config to the Back end nodes (Postgresql)
+  - create a config file `postgresql.toml`
+
+``` cmd
+    chef-automate config patch postgresql.toml --postgresql
+```
+sorthands for --postgresql is --pg and -p
+
+{{< note >}}
+
+- Frontend patch will be applied to all nodes where are Postgresql and OpenSearch changes will be applied to only one node
```

Comment:
- Frontend patch will be applied to all nodes where Postgresql and OpenSearch changes will be applied to only one cluster node.